test/unit/codegeneration/TempVarTransformerUseLet.js

```diff
@@ -0,0 +1,60 @@
+// Copyright 2014 Traceur Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {suite, test, assert} from '../../unit/unitTestRunner.js';
+import {Compiler} from '../../../src/Compiler.js';
+
+suite('TempVarTransformerUseLet.js', function() {
+
+  const USE_VAR = true;
+  const USE_LET = 'parse';
+
+  function testResult(blockBinding, content, expectedResult) {
+    test('', function() {
+      let compiler = new Compiler({
+        blockBinding,
+        script: true,
+        sourceMaps: false
+      });
+      let result = compiler.compile(content);
+      assert.equal(result, expectedResult);
```

Comment:
Maybe we should just check the first three characters? The test should verify that we emitted `let`/'var` when we should emit `let`/`var`.  If the test fails because of a mismatch in other parts of the expected output, it won't be because of `TempVarTransformer`.